src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+body,
+.nav {
   margin: 0;
+  padding: 0;
+  max-width: 1200px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.nav {
+  justify-content: space-between;
+  display: flex;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.nav_link {
+  font-family: Roboto, sans-serif;
```

Comment:
Move it to html selector

Reply:
can you explain please?

Reply:
goof practise to set font-family in html selector